### DISCUSSION TITLE
Style Number Measures

### DISCUSSION THREAD

**Discussion by gerardoscoppetta** [opensheetmusicdisplay/opensheetmusicdisplay#1479] [Q&A]
it is possible to change the color of the number of measures individually and also change the font size. Thank you.

![Number Measures](https://github.com/opensheetmusicdisplay/opensheetmusicdisplay/assets/41266915/29561494-9576-45db-a1da-034b0c4b6704)

**Comment by sschmidTU** [comment]
@gerardoscoppetta Hi, yes, you can get the SVG node of the measure number label and manipulate it to your heart's content, which should be instant (faster than re-rendering).

To get the SVG node:
```js
osmd.GraphicSheet.MusicPages[0].MusicSystems[0].measureNumberLabels[0].SVGNode
```

To e.g. change the color to blue:
```js
osmd.GraphicSheet.MusicPages[0].MusicSystems[0].measureNumberLabels[0].SVGNode.childNodes[0].setAttribute("fill", "#0000FF")
```
<img width="24" alt="image" src="https://github.com/opensheetmusicdisplay/opensheetmusicdisplay/assets/33069673/c0f50dbe-91dc-4cb8-9f44-10d610ee83ef">

Of course, you should build your own functions to get a particular system and measure number label by looping through the pages and systems, etc.

**Reply by gerardoscoppetta** [reply]
Great. It works great. I also tried with font-size. Thank you very much.
![Test color Number measure](https://github.com/opensheetmusicdisplay/opensheetmusicdisplay/assets/41266915/a5cd4e7a-1de8-4870-a147-6a24e70a6f9a)

**Reply by sschmidTU** [reply]
Great, good to know, thanks for the report!
